lib/linguist/languages.yml

```diff
@@ -619,6 +619,14 @@ Brainfuck:
   codemirror_mode: brainfuck
   codemirror_mime_type: text/x-brainfuck
   language_id: 38
+Brighterscript:
```

Comment:
is it officially "Brighterscript" or "Brighter<ins>S</ins>cript"?

Reply:
the latter actually, you are correct.

Reply:
i will update it today. thank you

Reply:
changed

Reply:
the samples folder will also need to be updated

Reply:
done, thanks

Reply:
Is there a pattern to have multiple possible syntaxes for the same extension? Some detection mechanism maybe.

Reply:
Yup, we have [heuristics](https://github.com/github/linguist/blob/master/lib/linguist/heuristics.yml) to differentiate. 

We don't need heuristics if the languages are sufficiently different as we can rely on the classifier to detect the language based on the tokens within the samples. In this case, it's best to have at least 2 samples from each language.

Reply:
i've added the heuristics regexes.
Should this be enough? 
I didn't add the "Bikeshed" language to the list, as i don't know if there is any syntax highlighting available for it.

Reply:
found the syntax highlighting for bikeshed as well, added the language as requested.